BuffTrigger.lua

```diff
@@ -94,8 +97,8 @@ do
 
     -- Test OwnOnly
     if (
-      data.ownOnly == true  and WeakAuras.myGUID ~= acEntry.unitCaster or
-      data.ownOnly == false and WeakAuras.myGUID == acEntry.unitCaster
+      data.ownOnly == true  and WeakAuras.myGUID ~= acEntry.casterGUID or
+      data.ownOnly == false and WeakAuras.myGUID == acEntry.casterGUID
     ) then
```

Comment:
I renamed acEntry.unitCaster to casterGUID, and added a new unitCaster that is the actual name.